src/python/foglamp/data_purge/sqlalchemy_insert.py

```diff
@@ -0,0 +1,67 @@
+"""
+Date: June 21 2017 
+Description: The following is the CREATE table and INSERT into table using SQLAlchemy
+            It is intended to be used for when showing how purge (currently) works example. 
+Lanague: Python + SQLAlchemy
+"""
+import sys
+import sqlalchemy
+import sqlalchemy.dialects
+import time
+
+# Set variables for connecting to database
+user="foglamp"
+db_user="foglamp"
+host="127.0.0.1"
+db="foglamp"
+
+# Create Connection
+engine = sqlalchemy.create_engine('postgres://%s:%s@%s/%s' % (db_user,user,host,db), pool_size=20, max_overflow=0)
+conn = engine.connect()
+
+# Important files
+configFile = 'config.json'
```

Comment:
@Terris-Dianomic - _CONFIG_FILE will be replaced by just a connection to the configuration.